meta.yml

```diff
@@ -9,29 +9,17 @@ description: "How federal government domains are meeting best practices on the w
 
 # Data sources. These URLs should have their content updated in-place.
 data:
-  # URL where we download the .gov domain list from.
-  domains_url: "https://github.com/GSA/data/raw/master/dotgov-domains/current-federal.csv"
+  # URL where we download the .ca domain list from.
+  domains_url: "./csv/domains.csv"
 
   # URL where we download the participating DAP .gov list from.
   analytics_url: "https://analytics.usa.gov/data/live/second-level-domains.csv"
 
   # DAP URL where we source public .gov subdomains from.
   analytics_subdomains_url: "https://analytics.usa.gov/data/live/sites-extended.csv"
 
-  # Preprocessed CSV where we source End of Term Archive data from.
-  eot_subdomains_url: https://github.com/GSA/data/raw/master/end-of-term-archive-csv/eot-2016-seeds.csv
-
-  # Snapshot of Censys.io data from late November 2017, interim while we
-  # re-establish use of their APIs.
-  censys_snapshot_url: https://github.com/GSA/data/raw/master/dotgov-websites/censys-federal-snapshot.csv
-
-  # Snapshot of Rapid7 Reverse DNS data from Scans.io in late 2017. The data
-  # is too big to automatically process each time with our current infrastructure,
-  # so this is relying on occasional snapshots which can be updated in-place in the repo.
-  rdns_snapshot_url: https://github.com/GSA/data/raw/master/dotgov-websites/rdns-federal-snapshot.csv
-
-  # Some other websites, collected and curated by GSA and DHS.
-  other_subdomains_url: https://github.com/GSA/data/raw/master/dotgov-websites/other-websites.csv
+  # Some other websites
+  other_subdomains_url: "./csv/subdomains.csv"
```

Comment:
Any particular reason you went with "other_subdomains_url" instead of something more descriptive?

Reply:
No particular reason, just that was what it already was and required less changing in env.py. Since we don't know the final structure of the CSVs we'll receive (ie. will we have more than one like 18F does) I figured just keep it as is until we know and can rename then :)